test/jdk/java/awt/TitledBorder/ScaledEtchedBorderTest.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Point;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.imageio.ImageIO;
+import javax.swing.BorderFactory;
+import javax.swing.Box;
+import javax.swing.BoxLayout;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8279614
+ * @summary The left line of the TitledBorder is not painted on 150 scale factor
+ * @requires (os.family == "windows")
+ * @run main TitledBorderTest
+ */
+
+public class ScaledEtchedBorderTest {
+
+    public static final Dimension SIZE = new Dimension(120, 20);
+
+    public static JFrame frame;
+    public static JPanel contentPanel;
+    public static JPanel parentPanel;
+    public static JPanel childPanel;
+    public static Color highlight = Color.RED;
+    public static Color shadow = Color.BLUE;
+
+    private static final double[] scales =
+            {1.00, 1.25, 1.50, 1.75, 2.00, 2.50, 3.00};
+
+    private static final List<BufferedImage> images =
+            new ArrayList<>(scales.length);
+
+    private static final List<Point> panelLocations =
+            new ArrayList<>(4);
+
+    public static void main(String[] args) throws Exception {
+        boolean showFrame = args.length > 1 && "-show".equals(args[0]);
+        try {
+            UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
+        } catch (Exception e) {
```

Comment:
Can we instead call
(https://docs.oracle.com/en/java/javase/17/docs/api/java.desktop/javax/swing/UIManager.html#getSystemLookAndFeelClassName())

Actually I'm not sure why only Win L&F is tested .. it is cross-platform and cross L&F code being changed

Reply:
> Actually I'm not sure why only Win L&F is tested .. it is cross-platform and cross L&F code being changed

It was exactly my point, we already [discussed it above](https://github.com/openjdk/jdk/pull/7449#discussion_r836586231).

The updated test uses `EtchedBorder` explicitly and it is not Look-and-Feel-specific, it can run on any platform.